arpack-ng-config.cmake.in

```diff
@@ -18,24 +17,9 @@ set(exec_prefix "@exec_prefix@")
 set(libdir "@libdir@")
 set(includedir "@includedir@")
 
-set(arpack_ng_INCLUDE_DIRS "${includedir}/arpack;${MPI_Fortran_INCLUDE_DIRS};${MPI_C_INCLUDE_DIRS};${MPI_CXX_INCLUDE_DIRS}")
-
-if(EXISTS "${libdir}/libarpack.a")
-  set(arpack_ng_LIBRARIES "${libdir}/libarpack.a")
-elseif(EXISTS "${libdir}/libarpack.so")
-  set(arpack_ng_LIBRARIES "${libdir}/libarpack.so")
-endif()
-
-if(EXISTS "${libdir}/libparpack.a")
-  set(parpack_ng_LIBRARIES "${libdir}/libparpack.a;${arpack_ng_LIBRARIES};${MPI_Fortran_LIBRARIES};${MPI_C_LIBRARIES};${MPI_CXX_LIBRARIES}")
-elseif(EXISTS "${libdir}/libparpack.so")
-  set(parpack_ng_LIBRARIES "${libdir}/libparpack.so;${arpack_ng_LIBRARIES};${MPI_Fortran_LIBRARIES};${MPI_C_LIBRARIES};${MPI_CXX_LIBRARIES}")
-endif()
-
 # Create arpack targets.
 add_library(ARPACK::ARPACK INTERFACE IMPORTED)
-set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${arpack_ng_INCLUDE_DIRS}")
-set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_LINK_LIBRARIES      "${arpack_ng_LIBRARIES}")
+set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${includedir}/arpack")
+set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_LINK_LIBRARIES      "arpack")
 add_library(PARPACK::PARPACK INTERFACE IMPORTED)
-set_target_properties(PARPACK::PARPACK PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${arpack_ng_INCLUDE_DIRS}")
-set_target_properties(PARPACK::PARPACK PROPERTIES INTERFACE_LINK_LIBRARIES      "${parpack_ng_LIBRARIES}")
+set_target_properties(PARPACK::PARPACK PROPERTIES INTERFACE_LINK_LIBRARIES    "parpack")
```

Comment:
Not sure to know if this will work: @turboencabulator , did you test ? https://github.com/opencollab/arpack-ng/pull/259#issuecomment-603829535